internal/nginx/templates/nginx-plus.tmpl

```diff
@@ -96,10 +96,10 @@ http {
 
         location  = /dashboard.html {
         }
-
-        allow 127.0.0.1;
+{{ range $value := .NginxStatusAllowCIDRs }}{{ if ne $value "" }}
```

Comment:
could you indent `{{ range ...` so that it has the indentation as the `allow` in the next line?